### DISCUSSION TITLE
TURKLERIN FESATLIGI

### DISCUSSION THREAD

**Discussion by mlkmutlu** [IBAX-io/go-ibax#948] [Soru Cevap - Türkçe]
SURADA BILE FESATIZ AQ.GIDIP HINTLILERE UKRAYNALILARA YORUM YAPIYORSUNUZ KENDI ULKENIZIN INSANI NE DEMIS BAKMIYORSUNUZ BILE YUH LAN UTANMAZLAR

**Comment by zukenberger** [comment]
helal bee be bro. Bari şu oka tıklasalar bari
